packages/microsoft-reactnative-sampleapps/index.windows.js

```diff
@@ -7,16 +7,16 @@ import React, { Component } from 'react';
 import {
```

Comment:
Was originally going to enforce sorted imports, but the change was pretty massive and behaves annoyingly with eslint. Can revert this file if wanted.